release-notes/DxMs/cloudgateway_change_log.md

```diff
@@ -4,6 +4,16 @@ uid: cloudgateway_change_log
 
 # Cloud Gateway change log
 
+#### 16 July 2024 - Fix - CloudGateway 2.13.14 - Requests could get no response in an edge case leading to timeouts [ID_40208]
+
+If an issue occurred while handling a remote access web API call, it could happen that no response was returned, which would lead to the call going into timeout. Now the CloudGateway will try to return an internal server response immediately.
+
+Also a warning log has been added if a remote access web API call would take longer than 25 seconds.
```

Comment:
If a remote access web API call takes longer than 25 seconds, this will now also be mentioned in a warning log.